db/seeds.rb

```diff
@@ -11,3 +11,18 @@
   avg_rating: 4,
   location: "Harvard Square"
 )
+
+FoodTruck.create(
+  name: "Banh Mi",
+  description: "Food truck that specializes in Asian fusion-type foods.",
+  avg_rating: 4,
+  location: "South Station"
+)
+
+FoodTruck.create(
+  name: "Mei Mei",
+  description: "Food truck that serves creative Chinese-American cuisine made
+  from locally sourced and sustainable ingredients.",
+  avg_rating: 4,
+  location: "Fenway"
+)
```

Comment:
It is good practice to write your seeder to allow for multiple runs. If a food truck name is unique, this will error out on the second run of `rake db:seed`. Check out the [lesson on seeding](https://learn.launchacademy.com/lessons/seeding)